### ISSUE TITLE
Can't find the "send to kindle" button

### ISSUE THREAD

**Issue by LoyerG** [janeczku/calibre-web#1858]
Hello, 

Before we start, thanks to all the developers working on this project, it's really great.

I recently created a web-caliber server on my Synology. I added the email of my Kindle to my account, I set up the mail server via my Hotmail address, and everything is working fine, the test email is well sent. I also configured my email address directly on my Amazon account on Device and Content>Preference>Personal Mail settings. 

In short, all my configuration seems to be correct, and despite everything, I can't find the "Send to Kindle" button anywhere. So my question is going to sound stupid, but where can I find this button? When I display a book's details, it is nowhere. 

Here is un screenshot og what i get when i open a book: 
![image](https://user-images.githubusercontent.com/12394684/107750034-ba2d2f00-6d1b-11eb-887c-e846635b4ad3.png)

**Comment by LoyerG** [comment]
Okay, I found the problem. It's been 4 days that I'm on it and I've solved my problem by myself, 5 minutes after creating an issue... Calibre-web allows sending only for mobi and azw formats.
